src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/aStar.java

```diff
@@ -0,0 +1,96 @@
+package edu.greenblitz.tobyDetermined.Nodesssss;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+import java.util.LinkedList;
+
+public class aStar {
+    static LinkedList<NodeArm> open = new LinkedList<>();
+    static LinkedList<NodeArm> closed = new LinkedList<>();
+    static NodeBase nodeBase = NodeBase.getInstance();
+
+   public static double getHGcost(NodeArm point, NodeArm current){
```

Comment:
Raz - in general in this file, explain what is H,F,G as part of the function name

Reply:
changed names of functions